Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/_FullPartial.cshtml

```diff
@@ -0,0 +1,131 @@
+﻿@model ContestResultsViewModel
+
+@using OJS.Common.Models
+@using OJS.Web.Areas.Contests.Controllers
+@using OJS.Web.Areas.Contests.ViewModels.Results
+@using OJS.Web.Areas.Users.Controllers
+
+@using Resource = Resources.Areas.Contests.Views.ResultsFull
+
+@{
+    const string ResultsContainerId = "full-results-container";
+    const string LoadingMaskContainerId = "full-results-loading-mask-container";
+    const string OnPageChangedEventHandlerName = "OnPageChanged";
+    var isOfficialStr = Model.IsCompete ? CompeteController.CompeteActionName : CompeteController.PracticeActionName;
+    var isOfficial = Model.IsCompete;
+    var results = Model.Results;
+}
+
+<div class="results-container" id="@ResultsContainerId">
+    @if (results.PageCount > 1)
+    {
+        @Html.PagedListPager(
+            results,
+            page => Url.Action<ResultsController>(
+                c => c.FullPartial(
+                    Model.Id,
+                    isOfficial,
+                    page,
+                    results.PageSize)),
+            PagedListRenderOptions.EnableUnobtrusiveAjaxReplacing(new AjaxOptions
+            {
+                HttpMethod = "GET",
+                UpdateTargetId = ResultsContainerId,
+                LoadingElementId = LoadingMaskContainerId,
+                OnSuccess = OnPageChangedEventHandlerName
+            }))
+    }
+
+    @if (results.Count > 0)
+    {
+        <script src="/Scripts/Helpers/test-results.js"></script>
```

Comment:
Scripts should not be in partial views, all scripts should be in the main view to ensure the same script will not be reimported twice if the partial view is imported twice.

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/223